corehq/apps/accounting/models.py

```diff
@@ -3376,6 +3377,14 @@ def get_filename(invoice):
             'month': invoice.date_start.month,
         }
 
+    @staticmethod
+    def can_pay_by_wire(invoice):
```

Comment:
can we clean this up with the other `can_pay_by_wire` options? maybe creating a utility too?

Reply:
Put this on the Invoice classes so it can be referenced by both the invoice emails and the invoice PDF. The implementation is slightly different for each type of invoice so there are still four different `can_pay_by_wire` properties. This actually would have caused a bug previously in trying to get `invoice.subscription` versus `invoice.subscriptions` from a `CustomerInvoice`, so I'm happy to have caught that while making this change. https://github.com/dimagi/commcare-hq/pull/36155/commits/0c6feef0a511495d47a8f65707def128b1fd1ce5